TypeCobol.Test/Parser/Programs/Cobol85/MultiLines.cbl

```diff
@@ -1,15 +1,19 @@
 ﻿ IDENTIFICATION DIVISION.
- PROGRAM-ID. PGM.
+ PROGRAM-ID. TCOMFL01.
  ENVIRONMENT DIVISION.
  CONFIGURATION SECTION.
  SOURCE-COMPUTER. IBM-370.
  OBJECT-COMPUTER. IBM-370.
  DATA DIVISION.
  WORKING-STORAGE SECTION.
- 01  MyGROUP.
-     05 VAR1 PIC X(78) value 'f
+ 01  GROUP1.
+     05 VAR1 PIC X(49) value 'f
 -       ' accès illimité* à vos comptes                  '.
-     05 VAR2 PIC X(46) value '
+     05 VAR2 PIC X(9) value '
 -        '         '.
  PROCEDURE DIVISION.
-     .
\ No newline at end of file
+     DISPLAY VAR1
+     DISPLAY VAR2
+     GOBACK
+     .
+ END PROGRAM TCOMFL01.
```

Comment:
Why not cover this in InitialValuesContinuations.rdz.cbl  ?

Reply:
This test is an adaptation of an earlier test that was left in the ``NewTestsToFix` directory.

Reply:
I agree with @rooksdo, it's better to factorize these 2 tests.

Reply:
See d922866c0f084e1fe573589d3089411bdaaa1b7b.